pdp/flags_mapper_pca_test.go

```diff
@@ -94,3 +97,58 @@ func TestFlagsMapperPCAOrdering(t *testing.T) {
 		}
 	}
 }
+
+const (
+	expectEmptyFlagMapperPCAJSON = `{"type":"flagsMapperPCA","def":"\"\"","err":"\"\"","alg":{"type":"firstApplicableEffectPCA"}}`
+	expectFlagMapperPCAJSON      = `{"type":"flagsMapperPCA","def":"\"first\"","err":"\"second\"","alg":{"type":"firstApplicableEffectPCA"}}`
+)
```

Comment:
Since these constants are only used in the func below, I would move them inside the func.